packages/docusaurus-plugin-content-docs/src/versions.ts

```diff
@@ -288,11 +288,9 @@ function createVersionMetadata({
   // retro-compatible values
   const defaultVersionLabel =
     versionName === CURRENT_VERSION_NAME ? 'Next' : versionName;
-  const defaultVersionPathPart = isLast
-    ? ''
-    : versionName === CURRENT_VERSION_NAME
-    ? 'next'
-    : versionName;
+  const versionNameNotLast =
```

Comment:
this name is a bit confusing. What about using a function like getDefaultVersionPathPart instead?